WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/MostPopularInsightsUseCase.kt

```diff
@@ -80,7 +80,8 @@ class MostPopularInsightsUseCase
         return items
     }
 
-    private fun buildTitle() = Title(R.string.stats_insights_popular, menuAction = this::onMenuClick)
+    private val fireEmojiTitle = "\uD83D\uDD25 ${resourceProvider.getString(R.string.stats_insights_popular)}"
```

Comment:
Do you think we should add the emoji to the string resources, so translators can also manage the emoji?

Reply:
I couldn't get it to work in strings yesterday.  Found another way that works today